src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react'
```

Comment:
I would also add a simple test for `RenderPropsComponents/DayOfWeek/DayOfWeek.tsx` to cover the main functionality (class addition basically)

Reply:
Well, it’s kind of testing implementation details, but okay, I’ll add a snapshot test

Reply:
done